lock-api/src/main/java/com/palantir/lock/client/metrics/TimeLockFeedbackBackgroundTask.java

```diff
@@ -51,7 +52,7 @@ public final class TimeLockFeedbackBackgroundTask implements AutoCloseable {
     private static final ScheduledExecutorService executor =
             PTExecutors.newSingleThreadScheduledExecutor(new NamedThreadFactory(TIMELOCK_FEEDBACK_THREAD_PREFIX, true));
 
-    private final UUID nodeId = UUID.randomUUID();
+    private final UUID nodeId = UniqueIds.randomUuidV4();
```

Comment:
used by timelock for identifying clients as part of feedback - can be pseudorandom, though given it's only called once, doesn't really matter?

Reply:
yeah, I figured this wasn't on hot path so shouldn't matter.